addons/web/static/src/core/commands/command_palette.variables.scss

```diff
@@ -0,0 +1,8 @@
+$o-command-palette-margin-top: 10vh;
+$o-command-palette-search-border: 1px solid rgba(0, 0, 0, 0.1);
```

Comment:
- This color is not consistent.
- This color is not automatically "translatable" to dark mode

Just use `$border-color`. Actually, just add a `border-bottom` class